### ISSUE TITLE
No data returned

### ISSUE THREAD

**Issue by sebelk** [architv/soccer-cli#139]
Hi, I'm using soccer-cli (0.1.0.0) and it returns no data, for example:

````
$ soccer --players --team=BM
No data for the team. Please check the team code.
````

No standings, no players, nothing at all. Is this project alive?

The python version is 3.6.7

Thanks in advance

**Comment by Saturn** [comment]
You should still be able to get some data.

Although, the data source changed its API which is why if you try to get standings for the English Premier League for example, you will get data from last season (2017/2018).

The code would need to be adjusted to support the new API. As well as potentially updated to support the various access tiers which include support for different leagues/statistics etc.

https://www.football-data.org/documentation/api

**Comment by tiktuk** [comment]
I don't get any data returned either.

    $ soccer --standings --league=EPL
    No standings availble for EPL.

**Comment by Saturn** [comment]
Okay, it seems like the old v1 api url no longer returns the "old" data either.

The new api is structurally a little different so it isn't totally simple to change over.

I will have a closer look soon.

Thanks,

**Comment by Saturn** [comment]
@tiktuk @sebelk I have updated most of the commands to work with the new version of the api. https://github.com/architv/soccer-cli/commit/7fdabcf8abb0b4a33c9bf98e6ba5e16a1cfde3d1

Are you able to get data back now?

**Comment by tiktuk** [comment]
Yes, thanks, it looks good ☺️ !

**Comment by dfrt82** [comment]
> Okay, it seems like the old v1 api url no longer returns the "old" data either.
Yes that's correct. v1 was shut down last week, but it was announced in July and December that it will be shut down to the end of 2018...

> The new api is structurally a little different so it isn't totally simple to change over.
Yes, but just a little :-) Major enhancement are the fixed league codes, so now there's no need to touch the code for the new season any more.
